tests/time_tests/time-testhelper/include/time-testhelper/utils.h

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
```

Comment:
This should have `#include <string>` and `namespace TimeTest {`

Reply:
Done